kuma/static/js/components/banners/utils/banners-state-util.js

```diff
@@ -0,0 +1,126 @@
+(function() {
+    'use strict';
+    /**
+     * Length of time a banner remains hidden after it's been disabled
+     * by clicking the close button.
+     * @const
+     */
+    var BANNER_DISABLED_EXPIRATION = 5 * 24 * 60 * 60 * 1000; // 5 days.
+    /**
+     * A simple string to use as a namespace for `localStorage` items
+     * @const
+     */
+    var LOCALSTORAGE_NAMESPACE = 'mdnBanner.';
+    var mdnBannersStateUtil = {
+        /**
+         * Filters the `bannerIds` down to only those in an active waffle flag state
+         * @returns {Array} of banner ids or an empty array if no active banners
+         */
+        getActiveWaffledBanners: function() {
```

Comment:
I like this, except that the word "active" is now overloaded to mean two different things. What if you called it getWaffleEnabledBanners()?